src/Cli/Commands/ExportOptions.cs

```diff
@@ -1,27 +1,63 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using Azure.DataApiBuilder.Core.Generator;
 using CommandLine;
 
 namespace Cli.Commands
 {
-    [Verb("export", isDefault: false, HelpText = "Export the GraphQL schema as a file and save to disk", Hidden = false)]
+    [Verb("export", isDefault: false, HelpText = "Export the GraphQL schema as a file and save it to disk.", Hidden = false)]
     public class ExportOptions : Options
     {
-        public ExportOptions(bool graphql, string outputDirectory, string? config, string? graphqlSchemaFile) : base(config)
+        public ExportOptions(bool graphql, string outputDirectory, string? graphqlSchemaFile,
+                bool? generate, string? samplingMode, int? numberOfRecords, string? partitionKeyPath, int? maxDays, int? groupCount, string? config) : base(config)
         {
             GraphQL = graphql;
             OutputDirectory = outputDirectory;
-            GraphQLSchemaFile = graphqlSchemaFile ?? "schema.graphql";
+            GraphQLSchemaFile = graphqlSchemaFile ?? "schema.gql";
```

Comment:
I know we don't have any existing tests for export command. But would be possible to add one since there are changes related to it ?

Reply:
lets have a separate PR to do test coverage for Cli project. This PR already have feature changes. lets not make it bigger :)